pandas/core/indexing.py

```diff
@@ -2320,6 +2314,12 @@ def _convert_key(self, key, is_setter: bool = False):
         return key
 
 
+def _tuplify(ndim: int, loc) -> tuple:
+    tup = [slice(None, None) for _ in range(ndim)]
```

Comment:
can you add a doc-string

Reply:
sure

Reply:
docstrings added.  will take another look at removing the discussed lines in the next pass